### PR TITLE
#213/create an endpoint to check if a user is blocked by the authenticated user

### LINKED ISSUE #210: Create a database method to read a block

### LINKED ISSUE #213: Create an endpoint to check if a user is blocked by the authenticated user
Create a `GET /users/block/{userId}` endpoint to check if a user is blocked by the authenticated user.

### PR DESCRIPTION
**Pull request by gbudau**
close #210 
close #213

### CODE REVIEW

**Review by vdedios — COMMENTED** [review]
Nice!

**Comment by Damrod** [comment]
It doesn't work for me:
Screenshot from 2022-10-26 15-53-32

File: transcendence-app/src/user/user.controller.ts [hunk 1]
```diff
@@ -222,4 +222,23 @@ export class UserController {
...
+    @GetUser() user: User,
+    @Param('userId') blockedUserId: string,
+  ): Promise<void> {
+    const block = await this.relationshipService.getBlock(
+      user.id,
+      blockedUserId,
+    );
+    if (!block) {
+      throw new NotFoundException();
+    }
```
Comment (Damrod) [inline transcendence-app/src/user/user.controller.ts:242]: I would assume it's not working because you are not returning anything from this.

Comment (Damrod) [reply transcendence-app/src/user/user.controller.ts:242]: It fits work, as discussed.

**Review by Damrod — APPROVED** [review]
Works! Thanks for this feature!

**Comment by gbudau** [comment]
Thanks!

**Pull request merged**